docs/public/docker.md

```
@@ -0,0 +1,313 @@
+      - caddy-data:/data
+      - ./Caddyfile:/etc/caddy/Caddyfile
+    restart: unless-stopped
+
+volumes:
+  kandev-data:
+  caddy-data:
+```
+
+Example `Caddyfile`:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)